ui/src/components/admin/Model/TabHeader.jsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { css } from '@emotion/react';
-import MomentReact from 'react-moment';
+// TODO: import MomentReact from 'react-moment';
```

Comment:
Have not successfully gotten this running, but only appears to be on the Admin side (and only handling dates).

Reply:
Fixed in [e897d3a](https://github.com/nci-hcmi-catalog/portal/pull/1123/commits/e897d3acb222f53337cb6a003d60e4f26ceac322) using a specific Vite config setting, added a comment for future reference